node-module-system/calculate.js

```diff
@@ -0,0 +1,17 @@
+import add from './add.js';
+import subtract from './subtract.js';
+import multiply from './multiply.js';
+import divide from './divide.js';
+
+if (process.argv[3] === 'minus') {
+  subtract(4, 2);
```

Comment:
These will always log the same value. That's not very useful for a general purpose calculator. Fix it so that it adds, subtracts, etc., using the numbers that were typed in on the command line (the command line arguments).